### DISCUSSION TITLE
Can't run on AWS Lightsail cp: cannot create regular file '/home/steam/valheim/linux64/': Not a directory

### DISCUSSION THREAD

**Discussion by mockingjet** [mbround18/valheim-docker#284] [Help!] [answered]
I am running into this issue

Follow this tutorial:  [AWS Lightsail](https://aws.amazon.com/tw/getting-started/hands-on/valheim-on-aws/)

![image](https://user-images.githubusercontent.com/32839503/110898105-96203600-8339-11eb-99d4-e5f69bab7260.png)

![image](https://user-images.githubusercontent.com/32839503/110895564-fbbdf380-8334-11eb-9345-760087a42141.png)

![image](https://user-images.githubusercontent.com/32839503/110895834-5c4d3080-8335-11eb-85fc-ae9448a57b75.png)

**Comment by mbround18** [comment]
It looks like it did not install? On your host, run `echo "$(id -u):$(id -g)"` and post your results here

**Reply by mbround18** [reply]
@mockingjet based on replies in this discussion but in another thread, are you using the recommendation Valheim server size?

![Screenshot_20210313-113338.jpg](https://user-images.githubusercontent.com/12646562/111042058-9cc1b280-83f8-11eb-98f6-d3f97378212a.jpg)

**Comment by rp199** [comment]
I'm getting the exact same error while following the same tutorial:

Here are my full logs (also tested with custom valus for Name and world):

![image](https://user-images.githubusercontent.com/52860444/111040040-25f0dd80-8429-11eb-8cb5-f2e1b30fdcda.png)

Already changed the docker compose to contain the right ids (this sample is the one without any customer change):

```
version: "3"
services:
valheim:
image: mbround18/valheim:latest
ports:
- 2456:2456/udp
- 2457:2457/udp
- 2458:2458/udp
environment:
- PORT=2456
- NAME="YOURSERVERNAME"
- WORLD="Dedicated"
- PASSWORD="ATLEAST5CHARACTERS"
- TZ=America/Chicago
- PUBLIC=1
- AUTO_UPDATE=1
- AUTO_UPDATE_SCHEDULE="0 1 * * *"
- AUTO_BACKUP=1
- AUTO_BACKUP_SCHEDULE="*/15 * * * *"
- AUTO_BACKUP_REMOVE_OLD=1
- AUTO_BACKUP_DAYS_TO_LIVE=3
- AUTO_BACKUP_ON_UPDATE=1
- AUTO_BACKUP_ON_SHUTDOWN=1
- PUID=1000
- PGID=998
volumes:
- ./valheim/saves:/home/steam/.config/unity3d/IronGate/Valheim
- ./valheim/server:/home/steam/valheim
- ./valheim/backups:/home/steam/backups
```
Am I missing something here?

**Reply by mbround18** [reply]
It looks like you are running with too little ram, what size are you running?

**Reply by rp199** [reply]
Yes just arrived at the same conclusion. I choose the smallest option 512MB. That's clearly not enough. I will try to run on 2GB, that's the minimum right?

**Reply by rp199** [reply]
Manage to get it started after increasing the ram to 2GB 🙂

**Reply by mbround18** [reply]
The tutorial recommends the 2 core 4gb variant.

![Screenshot_20210313-113338.jpg](https://user-images.githubusercontent.com/12646562/111042020-697f2380-83f8-11eb-83ff-57a49364f063.jpg)


2gb is the bottom of the barrel bare minimum and you will have to keep a sharp eye on crashes regarding your systems ram.

**Reply by rp199** [reply]
Yeah, I can confirm that 2gb is not enouth also. It looked like it was started but it turns out it didn't... Created a 4gb instance just now and was able to connect and play. This teached me to not try to be cheap next time 😅

**Comment by mbround18** [comment, marked as answer]
It looks like you don't have adequate server resources. Please upgrade your instance to the recommended size for lightsail 


![Screenshot_20210313-113338.jpg](https://user-images.githubusercontent.com/12646562/111042094-ced31480-83f8-11eb-8131-e87fe82abddd.jpg)
